static/js/app.js

```diff
@@ -165,16 +167,23 @@ $(function() {
     var AppRouter = Backbone.Router.extend({
         routes: {
             "" : "navigateEmpty",
-            "/?marker=:id&start_date=:start&end_date=:end&show_fatal=:showFatal&show_severe=:showSevere&show_light=:showLight&show_inaccurate=:showInaccurate&zoom=:zoom&lat=:lat&lon=:lon" : "navigate"
```

Comment:
Yes, the routing didn't work with standard URLs for some reason, so I switched to work with the hash  '#' URLs.